### ISSUE TITLE
Bug: Strange characters

### ISSUE THREAD

**Issue by rastaman111** [TortugaPower/BookPlayer#1100]
When I import a file I get strange characters even though the file itself has normal metadata. File attached

![Simulator Screenshot - iPhone 15 - 2024-02-15 at 22 20 12](https://github.com/TortugaPower/BookPlayer/assets/15727518/1795dfa5-4468-4e3e-936e-2735e11c2240)
![Simulator Screenshot - iPhone 15 - 2024-02-15 at 22 20 37](https://github.com/TortugaPower/BookPlayer/assets/15727518/cda2fc14-88d9-4560-85ae-f44520ee7ef8)
![Simulator Screenshot - iPhone 15 - 2024-02-15 at 22 20 02](https://github.com/TortugaPower/BookPlayer/assets/15727518/3fa6ab8e-a97b-4a89-92ec-651819bdbfdb)

[01. Мальчик, который выжил.mp3.zip](https://github.com/TortugaPower/BookPlayer/files/14301338/01.mp3.zip)

**Comment by GianniCarlo** [comment]
Hi @rastaman111 , thanks for attaching the file, I ran it through ffprobe to see the encoded metadata, and we can clearly see the strange characters in there

<img width="1040" alt="Screenshot 2024-02-15 at 15 16 46" src="https://github.com/TortugaPower/BookPlayer/assets/14112819/daea3d67-b707-48fe-b5d1-8840c4157866">

I'm closing the issue, but feel free to continue the discussion, as a side note, you can go into the item details screen to change the title and author

**Comment by rastaman111** [comment]
Yes, strange encoding, I ran it through several online readers and they showed strange characters, although the Music application shows normal text

I noticed that it uses the Lavf58.44.100 encoding format.
I took another file that uses the LAME3.98 encoding format and it is readable.

I think that the problem is Lavf58.44.100

<img width="1112" alt="Снимок экрана 2024-02-15 в 23 22 20" src="https://github.com/TortugaPower/BookPlayer/assets/15727518/2ef67a6f-6ae6-4efa-9939-8af73118eb26">

**Comment by GianniCarlo** [comment]
@rastaman111 interesting 🤔 thanks for the info, I always thought that the metadata was corrupted in some way whenever I saw similar reports about a file's metadata. If the Music app does show the correct string, maybe there's a way to handle it inside the app as well

**Comment by rastaman111** [comment]
I understand that this format refers to a third-party library that can read this FFmpeg encoding

**Comment by rastaman111** [comment]
Hello!
I also found this file, it belongs to the Audiobook type and therefore it does not have a commonKey "artist" in the method data, but there is a TPE2 key that is responsible for "artist" by type id3

Magic ;)

File attached

[02. The Vanishing Glass.mp3.zip](https://github.com/TortugaPower/BookPlayer/files/14315630/02.The.Vanishing.Glass.mp3.zip)
